### DISCUSSION TITLE
Backup selected PVCs only

### DISCUSSION THREAD

**Discussion by fad3t** [vmware-tanzu/velero#6865] [Community support Q&A] [answered]
Hello,

I'm currently taking daily backup of my cluster, which includes all the resources (manifests) as well as volume snapshots (via the CSI driver). There are a lot of volumes for which I don't actually need a snapshot, so I would like to filter those out.

Ideally, I would like to set a label on the PVCs that need to be backed up (e.g. `backup: true`). I've tried using the `selector` flag, but this applies to all resources, not only PVCs. Is there a clean way to do this?
I've looked at resources policies but haven't seen any example using labels so I'm unsure whether this is supported.

Another option would be to trigger 2 backups (one with all manifests but no snapshot, and another with only PVCs and the selector) but if that's doable with a single schedule that'd be cleaner and easier to manage.

Thanks in advance,
Fred

**Comment by draghuram** [comment, marked as answer]
Is this you by any chance?

https://kubernetes.slack.com/archives/C6VCGP4MT/p1694897546081979

**Reply by fad3t** [reply]
It is not 😄 but yeah that's exactly the same question/needs. I'll stick to 2 distinct schedules too then.

**Comment by newalexfl** [comment]
exactly the same question/needs. i want to takebackup for my site [haileys treasure apk](https://apkvent.com/haileys-treasure-adventure-apk/)
